.pre-commit-config.yaml

```diff
@@ -45,14 +45,12 @@ repos:
           - --autofix
 
   - repo: https://github.com/PyCQA/flake8
-    rev: 7.1.0
+    rev: 7.1.1
     hooks:
       - id: flake8
         additional_dependencies: [
-          flake8-blind-except,
           flake8-builtins,
           flake8-rst-docstrings,
-          flake8-logging-format,
```

Comment:
This looks like it broke in Python 3.12: https://github.com/pytest-dev/pytest-testinfra/issues/765

Reply:
I temporarly close this PR and then rebase it after #168 is merged, would that work?

Reply:
I don't think you need to close this one. Just merge in origin/master (and fix conflicts) once #168 is merged